### PR TITLE
Feature/jetpack connection flow install screen

### PR DESCRIPTION
**Pull request by planarvoid**
Replace Android dialog that was used to open the Jetpack connection flow with a new activity. The activity starts the flow and links to a more information FAQ page.

screenshot_1517564292

To test:
* Click on stats when Jetpack not installed/connected
* Setup page opens
* Setup button starts connection flow

### CODE REVIEW

**Comment by nbradbury** [comment]
In landscape the image and button are cut off.

screenshot_1517597724

**Comment by nbradbury** [comment]
As per the Slack conversation, we should remove the "More information" link since it takes you to a page explaining how to install Jetpack from your **browser**, which is out of place for the mobile app.

File: WordPress/src/main/java/org/wordpress/android/ui/stats/StatsConnectJetpackActivity.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package org.wordpress.android.ui.stats;
+
+import android.graphics.Paint;
+import android.os.Bundle;
+import android.os.PersistableBundle;
```
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/stats/StatsConnectJetpackActivity.java]: This import is unused.

**Comment by nbradbury** [comment]
After I tap "Set up Jetpack" I get taken to this mostly-empty web page.

screenshot_1517599009

**Comment by mattmiklic** [comment]
> In landscape the image and button are cut off.

Could we scale down the illustration and/or allow the screen to scroll on landscape?

Portrait orientation looks solid; makes sense re: removing the More Information link if it isn't mobile-specific.

File: WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java [hunk 2]
```diff
@@ -211,8 +211,10 @@ public void onClick(View v) {
                 if (!mAccountStore.hasAccessToken() && selectedSite != null && selectedSite.isJetpackConnected()) {
                     // If the user is not connected to WordPress.com, ask him to connect first.
                     startWPComLoginForJetpackStats();
-                } else {
+                } else if (selectedSite != null && selectedSite.isJetpackInstalled() && selectedSite.isJetpackConnected()){
                     ActivityLauncher.viewBlogStats(getActivity(), selectedSite);
+                } else {
+                    ActivityLauncher.startJetpackConnectionFlow(getActivity(), selectedSite);
```
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java]: Both the `if` statements above check for a null `selectedSite`. Should that be done here, too?

Comment (planarvoid) [reply WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java]: I have the null checks there to never have NPEs, not sure if null site is a valid state. I can totally add the check

**Comment by planarvoid** [comment]
@mattmiklic I remote the More Info button and added scrolling to landscape
@nbradbury as discussed, the Nexus 4 issue was passed to another team

**Comment by mattmiklic** [comment]
Sounds good @planarvoid. If you'd like to create a build or screenshots for me to look at I'm happy to give it a review, otherwise I can take a look once it's merged and give feedback on the landscape view.

**Pull request merged**